### PR TITLE
Add Docker quick start guide

### PR DESCRIPTION
**Pull request by mofirouz**

### CODE REVIEW

File: docs/install-docker-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+```csharp fct_label="Unity"
+INClient client = new NClient.Builder("defaultkey")
+    .Host("127.0.0.1")
+    .Port(7350)
+    .SSL(false)
+    .Build();
+```
+
+## Running Without Docker-Compose
+
```
Comment (AlimJaffer) [inline docs/install-docker-quickstart.md]: Maybe clarify exactly what Docker-Compose is here?   you've listed the positive trade-off, but if it's a true quickstart guide for someone who is new to Docker then maybe a one liner on it?

File: docs/install-binary.md [hunk 2]
```diff
@@ -1 +1,152 @@
...
+!!! note "CockroachDB Alternative"
+    Nakama unofficially supports [PostgreSQL](https://www.postgresql.org/download/) for environments where CockroachDB is not available.
+
+    For production settings, Nakama should be used with CockroachDB as queries are optimised due to the way CockroachDB stores data on the disk.
+
+## Install Nakama on MacOS
+
+You’ll need to be running MacOS 10.9 (Mavericks) to run Nakama.
+
+### Homebrew
```
Comment (AlimJaffer) [inline docs/install-binary.md]: are we still going to support homebrew per our discussions late last week?

File: docs/install-binary.md [hunk 3]
```diff
@@ -1 +1,152 @@
...
+```shell fct_label="Powershell"
+# migrate schema
+nakama.exe migration up
+# start the server and output logs to the terminal
+nakama.exe --log.stdout
+```
+
+## Install Nakama on Linux
+
+Using [Docker](install-docker-quickstart.md) - This is the recommended approach. We will soon provide packages for various Linux package managers to ease the installation procedure.
```
Comment (AlimJaffer) [inline docs/install-binary.md]: We could add a line here such as "If you have requests for specific package managers, please let us know so we can prioritise them"  - bit of a call to action

**Comment by AlimJaffer** [comment]
@mofirouz  - great stuff again.

File: docs/install-binary.md (hunk 2 above)
Comment (mofirouz) [inline docs/install-binary.md]: We can leave it here, and if anyone wants to try it, they can message us. 

Most of the focus is on Docker anyways :)

**Pull request closed without merging**